Flow.Launcher.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -145,13 +143,25 @@ public static bool FileExists(this string filePath)
             return File.Exists(filePath);
         }
 
+        /*
+         ❓ 
+        todo does it make sense to remove Flow.Launcher.Plugin.IPublicAPI.OpenDirectory (used as Context.API.OpenDirectory(path)),
```

Comment:
I think this is legacy...That one respect the setting and allow more plugins to utilize the custom explorer feature

Reply:
so what you're suggesting is to remove `FilesFolders.OpenPath`, correct?

Reply:
Added deprecation notice.